pegasus/cache/i18n/id-ID.yml

```diff
@@ -2191,11 +2191,3 @@
     Account” option:'
   hero_name: 'Minecraft: Hero''s Journey'
   applab-intro_name: App Lab Intro
-  homepage_below_hero_announcement_text: We care about your privacy and are making
```

Comment:
How come these strings are being removed from the pegasus files?

Reply:
Huh, I'm not sure

Reply:
Oh, likely because we need another sync up. I'll get on it